docs/tutorial/using-checkbox/test-output.rst

```diff
@@ -0,0 +1,80 @@
+.. _test_output:
+
+============================
+Test output
+============================
+
+When Checkbox runs the tests, it generates a series of outputs, which include the ID and category of the tests, along with the results of each them (if a test ID fails, the cause for the failure would be displayed). 
+
+The test output can be generally divided into the following sections:
```

Comment:
I would restructure this in something like: run the audio test and follow along. And then you can interleave the test plan output with explanations